Makefile

```diff
@@ -5,22 +5,10 @@ check-all: ## Run all lint checks and unittest
 	@echo "[Notice] If you'd like to run commands with same env to CI, please run \`tox\`."
 	@bash ci.sh
 
-.PHONY: isort
-isort: ## Run isort
-	python -m isort --profile black $(ARGS) .
-
-.PHONY: black
-black: ## Run black
-	python -m black $(ARGS) . -l 120 -S
-
-.PHONY: pylint
-pylint: ## Run pylint
-    # TODO Remove --disable=E1136 when no errors in py39
-	python -m pylint $(ARGS) --load-plugins pylint_quotes --rcfile .pylintrc appium test --disable=E1136
-
-.PHONY: mypy
-mypy:  ## Run mypy
-	python -m mypy appium test/functional
+.PHONY: ruff
```

Comment:
maybe create two steps: format + lint for local dev and the step `check`, which would be called automatically in the precommit hook

Reply:
the idea is that we don't want to rely on tool names, but rather intention purposes

Reply:
makes sense